adapters/characters_repository.go

```diff
@@ -0,0 +1,67 @@
+package adapters
+
+import (
+	"bootcamp/domain/model"
+	"bootcamp/infra/datastore"
+	"bootcamp/usecase/interfaces"
+	"fmt"
+	"strconv"
+	"strings"
+)
```

Comment:
Imports tend to follow this order by convention, each section separated by a newline.
```
1. Native packages.
2. External packages.
3. Internal packages.
```

[Here](https://stackoverflow.com/a/38714480/4521547)'s an example. The `goimports` tool can help you to do this automatically.

Reply:
I am using goimports and the imports get sorted in alphabetical order :/

Reply:
The example mentions a `-local` flag. Have you tried that?